src/shared/getComponentOption.js

```diff
@@ -64,6 +66,10 @@ export default function getComponentOption (opts, result = {}) {
 
       return metaObject
     })
+    result.meta = uniqBy(
+      result.meta.reverse(),
+      metaObject => metaObject.hasOwnProperty(tagIDKeyName) ? metaObject[tagIDKeyName] : uniqueId()
```

Comment:
I had to implement a workaround for tags without `vmid` set at all (which is common if you want to multiple tags of the same type)

My first idea was to use `Symbol` but it is not supported in IE :(

So I had to use another lodash fn